seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -1084,7 +1084,7 @@
         "/* head */ 'use strict'; /* solution */ var myStr = '<a href=\"http://www.example.com\" target=\"_blank\">Link</a>'; /* tail */ (function() { return \"myStr = \" + myStr; })();"
       ],
       "tests": [
-        "assert(!/\\\\/g.test(code) && myStr === '<a href=\"http://www.example.com\" target=\"_blank\">Link</a>', 'message: Remove all the <code>backslashes</code> (<code>\\</code>)');",
+        "assert(!/\\\\/g.test(code) && myStr.match('\\\\s*<a href=\"http://www.example.com\"\\\\s*target=\"_blank\">\\\\s*Link\\\\s*</a>\\\\s*'), 'message: Remove all the <code>backslashes</code> (<code>\\</code>)');",
```

Comment:
I agree that spaces should be allowed around the `=`.  The original code posted for this issue 
`var myStr ='<a href="http://www.example.com"  target= "_blank">Link</a>';` will still fail the test due to that extra whitespace after `target=`.